### PR TITLE
readme and screenshot

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob <jakob.roehrl@web.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+-   **📸 Beautiful Photo und Video Timeline**
+-   **⭐ Favorites and Tagging:** Favorite and tag your photos!
+-   **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+-   **Albums:** Create albums from your contents
+
+    Future plans:
+-   
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu.Consider installing the preview generator for pregenerating the thumnailsPlease have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.
```
Comment (jancborchardt) [inline README.md]: Some missing spaces and dots at end of sentences here in the middle. :)

File: README.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
...
+##Maintainers:
+- [John Molakvoæ](https://github.com/skjnldsv)
+If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)
+    
+## 🏗 Development setup
+    1.  ☁ Clone this into your `apps` folder of your Nextcloud
+    2.  👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies
+    3.  🏗 Then to build the Javascript whenever you make changes, run `make build-js`
+    4.  ✅ Enable the app through the app management of your Nextcloud
+    5.  🎉 Partytime!
```
Comment (jancborchardt) [inline README.md]: These don’t need to be indented, right?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
-3. 🏗 Then to build the Javascript whenever you make changes, run `make build-js`
-4. ✅ Enable the app through the app management of your Nextcloud
-5. 🎉 Partytime!
+![](screenshots/screenshot1.png)
+
+## Why is this so awesome?
+-   **📸 Beautiful Photo und Video Timeline**
+-   **⭐ Favorites and Tagging:** Favorite and tag your photos!
+-   **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+-   **Albums:** Create albums from your contents
```
Comment (jancborchardt) [inline README.md]: Indentation here is also a bit much, just needs to be 1 space after the dash.

File: README.md (hunk 1 above)
Comment (jancborchardt) [reply README.md]: Also would be good to link the preview generator here? :)

File: README.md [hunk 4]
```diff
@@ -1,7 +1,30 @@
-# Vue example app for Nextcloud
+# Photos
```
Comment (jancborchardt) [inline README.md]:
```suggestion
# Nextcloud Photos
```
That’s a bit nicer here :)

**Comment by jancborchardt** [comment]
Nice @jakobroehrl! See comments, and can you cut the screenshot to just the Nextcloud parts, no browser, OS or margins? :)

**Comment by jakobroehrl** [comment]
> 
> 
> Nice @jakobroehrl! See comments, and can you cut the screenshot to just the Nextcloud parts, no browser, OS or margins? :)

Yes, sure. Thanks for the review.

**Comment by skjnldsv** [comment]
Bump?

File: README.md [hunk 5]
```diff
@@ -1,7 +1,31 @@
...
+- **📸 Beautiful Photo und Video Timeline**
+- **⭐ Favorites and Tagging:** Favorite and tag your photos!
+- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+- **Albums:** Create albums from your contents
+    
+    Future plans:
+- ...
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails.
```
Comment (Mikescops) [inline README.md]: Should be `« Apps »`

File: README.md [hunk 6]
```diff
@@ -1,7 +1,31 @@
...
+Please have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.
+    
+## Mobile Photos
+Use the Android/iOS App to view your photos. It's possible to autoupload them.
+
+##Maintainers:
+- [John Molakvoæ](https://github.com/skjnldsv)
+If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)
+    
+## 🏗 Development setup
```
Comment (Mikescops) [inline README.md]: Would you mind adding an explanation for the linters? 
So everyone can lint and easily fix linting issues.
Also for vscode user we can add some tips for extensions.

File: README.md (hunk 5 above)
Comment (Mikescops) [inline README.md]: `thumnails` > `thumbnails`

File: README.md [hunk 7]
```diff
@@ -1,7 +1,31 @@
...
+    
+    Future plans:
+- ...
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails. 
+Please have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.
+    
+## Mobile Photos
+Use the Android/iOS App to view your photos. It's possible to autoupload them.
```
Comment (Mikescops) [inline README.md]: `auto-upload`

File: README.md [hunk 8]
```diff
@@ -1,7 +1,31 @@
...
+
+##Maintainers:
+- [John Molakvoæ](https://github.com/skjnldsv)
+If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)
```
Comment (Mikescops) [inline README.md]: wrong link >> `https://github.com/nextcloud/photos/issues`

File: README.md [hunk 9]
```diff
@@ -1,7 +1,31 @@
...
+    
+## 🏗 Development setup
+1.  ☁ Clone this into your `apps` folder of your Nextcloud
+2.  👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies
+3.  🏗 Then to build the Javascript whenever you make changes, run `make build-js`
```
Comment (Mikescops) [inline README.md]: To watch, the command is `make watch-js`

File: README.md (hunk 5 above)
Comment (jancborchardt) [reply README.md]: That’s only the french way. ;)

Let’s just go for:

> In your Nextcloud, go to the Apps management and enable the Photos app.

Comment (Mikescops) [reply README.md]: Fine ^^

File: README.md (hunk 6 above)
Comment (jakobroehrl) [reply README.md]: Do you mean adding the command "npm run lint:fix"?

Comment (Mikescops) [reply README.md]: Yep

**Comment by jakobroehrl** [comment]
@Mikescops 
Would you check again, please?

File: README.md [hunk 10]
```diff
@@ -1,7 +1,32 @@
...
+- **📸 Beautiful Photo und Video Timeline**
+- **⭐ Favorites and Tagging:** Favorite and tag your photos!
+- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+- **Albums:** Create albums from your contents
+    
+    Future plans:
+- ...
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumbnails.
```
Comment (Mikescops) [inline README.md]: »Apps« still not fixed should be `Apps`

Comment (jancborchardt) [reply README.md]:
```suggestion
In your Nextcloud, simply enable the Photos app through the Apps management.

Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating thumbnails. 
```

**Comment by Mikescops** [comment]
Can we move forward on this one before the release?

**Comment by Mikescops** [comment]
@jancborchardt 
image

**Comment by jancborchardt** [comment]
@Mikescops feel free to fix it, and also please add yourself to the maintainer list! :)

**Comment by Mikescops** [comment]
@jancborchardt sure, i'll take time tomorrow!

**Pull request merged**